package.json

```diff
@@ -29,9 +29,9 @@
     "validate-npm-package-name": "^3.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^16.2.1",
-    "@commitlint/config-conventional": "^16.2.1",
-    "@commitlint/config-lerna-scopes": "^16.2.2",
+    "@commitlint/cli": "16.3",
+    "@commitlint/config-conventional": "16.2.4",
+    "@commitlint/config-lerna-scopes": "16.2.4",
```

Comment:
It's fine to keep the caret in the version ranges

Reply:
@PaulRBerg Have added the caret. Please review again.